WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -108,6 +166,36 @@ private extension AddAttributeOptionsViewModel {
 
         return Section(header: Localization.headerSelectedTerms, footer: nil, rows: rows, allowsReorder: true)
     }
+
+    func createAddedSection() -> Section? {
+        // TODO: Handle attribute local options
+        guard state.optionsAdded.isNotEmpty else {
+            return nil
+        }
+
+        let rows = state.optionsAdded.map { term in
+            AddAttributeOptionsViewModel.Row.existingTerms(name: term.name)
+        }
+
+        return Section(header: Localization.headerExistingTerms, footer: nil, rows: rows, allowsReorder: false)
+    }
+
+    /// Synchronizes options(terms) for global attributes
+    ///
+    func synchronizeOptions() {
```

Comment:
nit: since this method synchronize just global attributes, maybe renaming it to `synchronizeGlobalOptions` (or `synchronizeGlobalTerms`)  it's more clear